ui/components/multichain-accounts/multichain-accounts-tree/multichain-accounts-tree.test.tsx

```diff
@@ -121,6 +121,30 @@ const mockWallets: ConsolidatedWallets = {
             },
             label: '',
           },
+          {
+            address: '0xC5b2b5ae370876c0122910F92a13bef85A133E56',
+            id: 'account-4',
+            metadata: {
+              name: 'Account 4',
+              keyring: {
+                type: 'HD Key Tree',
+              },
+              importTime: 0,
+            },
+            options: {},
+            methods: ETH_EOA_METHODS,
+            scopes: [EthScope.Eoa],
```

Comment:
Nit: But `Erc4337` accounts should not use `Eoa` scope, given that they cannot be available on "every EVM chains" by default (this as no impact on the test itself, but it might makes things more "valid")

NOTE: account 3 has the same problem (in this wallet)